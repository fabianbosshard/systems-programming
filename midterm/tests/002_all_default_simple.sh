exec "$1" < tests/lorem.txt
