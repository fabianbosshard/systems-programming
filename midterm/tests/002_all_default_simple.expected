Lorem
eiusmod
minim
aliquip
reprehenderit
pariatur.
culpa
